manifests/g/GyDi/ClashVerge/0.0.25/GyDi.ClashVerge.installer.yaml

```diff
@@ -9,9 +9,6 @@ Scope: machine
 InstallerSwitches:
   InstallLocation: INSTALLDIR="<INSTALLPATH>"
 UpgradeBehavior: install
-Dependencies:
-  PackageDependencies:
-  - PackageIdentifier: Microsoft.EdgeWebView2Runtime
```

Comment:
Why? During my test the installer still installs WebView2 explicitly. 🤔
![image](https://user-images.githubusercontent.com/56779163/159396909-4ad61da8-4ed7-4b5d-a392-3dde93e389c4.png)

Reply:
I’ve started to go back a few steps for dependencies that the installer installs itself, mainly because of the vc++2015-2919/2015-2022 bs